module/solutions/helper.js

```diff
@@ -2893,11 +2895,9 @@ module.exports = class SolutionsHelper {
 							referenceFrom: CONSTANTS.common.LINK,
 							link: link,
 						}
-						let checkForProjectExist = await projectQueries.projectDocument(
```

Comment:
@Prajwal17Tunerlabs what is this change. can you explain?

Reply:
while making a db query, earlier the code was passing token, which is not needed, and would throw error, so i have removed token from the function call